apps/wallet/src/background/account-sources/index.ts

```diff
@@ -72,6 +73,36 @@ async function createAccountSource({
 	}
 }
 
+async function deleteAccountSourceByType({
+	type,
+}: MethodPayload<'deleteAccountSourceByType'>['args']) {
+	switch (type) {
+		case 'mnemonic':
+			try {
+				const mnemonicAccountSources = await getAccountSources({ type: 'mnemonic' });
```

Comment:
we can lean on Dexie to reduce the amount of work we're doing here: 

```ts
const db = await getDB();
const sourceIds = await db.accountSources
	.where('type')
	.anyOf(['mnemonic', 'imported'])
	.primaryKeys();
const accountIds = await db.accounts.where('sourceID').anyOf(sourceIds).primaryKeys();

await db.accounts.bulkDelete(accountIds);
await db.accountSources.bulkDelete(sourceIds);
```
should accomplish the same thing